src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -56,6 +56,13 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('sitemap')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('dump_dir')->defaultValue('%sulu.cache_dir%/sitemaps')->end()
+                    ->scalarNode('default_host')->defaultNull()->end()
```

Comment:
i would prefer the prepend-extension because it could be that we have to had other values for that on different places (as we see this is all quite obscure because we don' t know it). i would go with this config. when we decide to create a "central" config we would have to place it in the core-bundle where we bootstrap the webspace-manager. but we have decided that we will "remove" it earlier or later and the other thing is that there will be a config which wont be needed in this bundle. i dont like it ...